src/test/java/core/basesyntax/service/strategy/BalanceHandlerTest.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.service.strategy;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+import core.basesyntax.db.Storage;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class BalanceHandlerTest {
+    private static final String FRUIT = "apple";
+    private static final int QUANTITY = 20;
+    private final Storage storage = new Storage();
+    private BalanceHandler handler;
+
+    @BeforeEach
+    void setUp() {
+        handler = new BalanceHandler(storage);
+    }
+
+    @Test
+    void put_informationToStorage_Ok() {
+        assertNull(storage.getData().get(FRUIT),
+                "Fruit should not exist in storage before operation.");
```

Comment:
just clear it before each test and we can remove that check from all handler tests